documentation/blog/2025-05-09-developers-ai-playbook-for-team-efficiency/index.md

```diff
@@ -0,0 +1,314 @@
+---
+title: "Championship Driven Development: Your Team's AI Playbook for Peak Performance"
+description: How AI-powered 'plays' can transform your dev team into a high-scoring sports team, streamlining game plans for debugging, changelogs, PRs, and more.
+authors:
+    - ian
+---
+
+![blog cover](cdd-playbook.png)
+
+# A Developer's Playbook: Using AI for Team Efficiency
+
+Development teams can operate like sports teams. Each member has a role and a shared playbook helps coordinate efforts. Let's explore how AI-driven "plays" can form a starter "playbook" for your dev team, helping with common technical tasks. You can use these "recipes" with [Goose](/) to leverage the Model Context Protocol (MCP) to make you more productive.
```

Comment:
Maybe take recipes out of quotes because it's not the fake name..it's the real name?